modules/nextflow/src/test/groovy/nextflow/script/ScriptBindingTest.groovy

```diff
@@ -160,68 +124,17 @@ class ScriptBindingTest extends Specification {
         then:
         copy.foo == 1
         copy.alphaBeta == 4
-        copy.'alpha-beta' == 4
         copy.delta == 2
         copy.gamma == 3
         copy.omega == 9
         and:
         // source does not change
         map.alpha == 0
         map.alphaBeta == 0
-        map.'alpha-beta' == 0
         map.delta == 2
         map.gamma == 3
         !map.containsKey('omega')
 
     }
 
-    def 'should wrap a string value with quote character' () {
-
-        expect:
-        ScriptBinding.ParamsMap.wrap('hello',null) == 'hello'
-        ScriptBinding.ParamsMap.wrap('hello','"') == '"hello"'
-        ScriptBinding.ParamsMap.wrap('hello world',null) == '"hello world"'
-        ScriptBinding.ParamsMap.wrap('hello world',"'") == "'hello world'"
-        ScriptBinding.ParamsMap.wrap('hello"world',"'") == "'hello\"world'"
-        ScriptBinding.ParamsMap.wrap('hello"world',null) == '"hello\\"world"'
-
-    }
-
-    def 'should return a command line formatted string'() {
-
-        when:
-        def params = new ScriptBinding.ParamsMap('foo-bar':1)
-        then:
-        params.size() == 2
-        params.fooBar == 1
-        params.'foo-bar' == 1
-        params.all() == '--foo-bar 1'
-
-        expect:
-        new ScriptBinding.ParamsMap(x:1).all() == '--x 1'
-        new ScriptBinding.ParamsMap(x:1, y: 2).all() == '--x 1 --y 2'
-        new ScriptBinding.ParamsMap(x:1, y: 2).all(sep:'=') == '--x=1 --y=2'
-        new ScriptBinding.ParamsMap(x:1, y: 2).all(sep:'=', prefix:'-') == '-x=1 -y=2'
-        new ScriptBinding.ParamsMap(x:1, y: 'hello world').all() == '--x 1 --y "hello world"'
-        new ScriptBinding.ParamsMap(x:1, y: 'hello world').all(quote:"'") == '--x \'1\' --y \'hello world\''
-        new ScriptBinding.ParamsMap(x:1, y: "O'Connors").all(quote:"'") == "--x '1' --y 'O\\'Connors'"
-
-    }
-
-    def 'should get the variable name giving the value'() {
-
-        given:
-        def X=1
-        def Y='hello'
-        def binding = new ScriptBinding([:])
-        binding.setVariable('foo', X)
-        binding.setVariable('bar', Y)
-
-        expect:
-        binding.getVariableName(X) == 'foo'
-        binding.getVariableName(Y) == 'bar'
-        binding.getVariableName(new String(Y)) == null
-        binding.getVariableName(null) == null
```

Comment:
Any reason this was deleted?

Reply:
weird, I'm not sure. maybe some mistake in resolving a merge conflict